package.json

```diff
@@ -7,15 +7,24 @@
   "author": "Marc Robert Wong <marcrobertwong@gmail.com>",
   "license": "MPL-2.0",
   "dependencies": {
-    "eslint-plugin-mozilla": "^0.3.4"
+    "clipboardy": "^1.1.4",
+    "eslint-plugin-mozilla": "^0.3.4",
+    "fs-promise": "^2.0.3",
+    "fx-runner": "^1.0.6",
+    "geckodriver": "^1.7.1",
+    "mocha": "^3.4.2",
+    "path": "^0.12.7",
+    "selenium-webdriver": "^3.4.0"
   },
   "devDependencies": {
     "eslint": "^4.0.0",
     "eslint-config-airbnb-base": "^11.2.0",
     "eslint-plugin-import": "^2.5.0"
   },
   "scripts": {
-    "lint": "eslint bootstrap.js",
-    "build": "./bin/make_xpi.sh"
+    "postinstall": "./bin/patch_webdriver.sh",
+    "lint": "eslint .",
+    "build": "./bin/make_xpi.sh",
+    "test": "export FIREFOX_BINARY=firefox && npm run build && XPI_NAME=extension/share-button-study.xpi mocha test"
```

Comment:
You're mixing two different styles of setting envvars here, using both export and also inline bars. You probably should use just one.